### PR TITLE
Nonconnected content

### PR DESCRIPTION
**Pull request by emileighoutlaw**
This commit adds:

- Updates landing page with nonconnected committee content
- Adds checklist page for nonconnected committees
- Updates termination page link for all essentials checklist pages (we now have a unified termination page)
- Links to nonconnected content from other checklist pages

### CODE REVIEW

**Comment by emileighoutlaw** [comment]
cc @noahmanger I think this is ready for review now!

**Comment by emileighoutlaw** [comment]
Arg! Thank you for those fixes, @noahmanger

File: fec/home/templates/home/nonconnected_checklist_page.html [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            <h2>How to file</h2>
+            <ul class="list--checks list--checks--secondary t-sans">
+              <li>Electronic filing
+                <ul>
+                  <li><a href="/registration-and-reporting/mandatory-electronic-filing/">Mandatory electronic filing</a></li>
+                  <li><a href="/registration-and-reporting/file-electronically/">Electronic filing overview</a></li>
+                  <li><a href="/registration-and-reporting/get-e-filing-password/">Getting an e-filing password</a></li>
+                 </ul>
+              </li>
+              <li><a href="/registration-and-reporting/party-committee-file-paper/">Paper filing</a></li>
```
Comment (noahmanger) [inline fec/home/templates/home/nonconnected_checklist_page.html]: This should be a nonconnected link, right?

Comment (emileighoutlaw) [inline fec/home/templates/home/nonconnected_checklist_page.html]: You're right—let me make a fix right now.

**Pull request merged**